cai_causal_graph/causal_graph.py

```diff
@@ -171,8 +171,19 @@ def is_edge_by_pair(self, pair: PAIR_T) -> bool:
         validate_pair_type(pair)
         return self.edge_exists(pair[0], pair[1])
 
+    def get_neighbors(self, node: NodeLike) -> List[str]:
+        """Get node identifiers for all neighbor nodes for a specific node."""
+        node_id = Node.identifier_from(node)
+        assert node_id in self.get_node_names(), f'Node with identifier {node_id} does not exist in this skeleton.'
+        return list(self.to_networkx().neighbors(node_id))
```

Comment:
I don't believe it's necessary to convert to networkx. Isn't that relatively expensive?

There's actually this function that I wrote: `cai_causal_discovery.utils.graph_utils.get_neighbors_in_mixed_graph` that should do the trick as well and does not convert to networkx. Can just copy?